sbt

```diff
@@ -277,10 +273,44 @@ acquire_sbt_jar () {
     [[ -r "$sbt_jar" ]]
   } || {
     sbt_jar="$(jar_file "$sbt_version")"
-    download_url "$(make_url "$sbt_version")" "$sbt_jar"
+    jar_url="$(make_url "$sbt_version")"
+
+    echoerr "Downloading sbt launcher for ${sbt_version}:"
+    echoerr "  From  ${jar_url}"
+    echoerr "    To  ${sbt_jar}"
+
+    download_url "${jar_url}" "${sbt_jar}"
+
+    case "${sbt_version}" in
+      0.*) vlog "SBT versions < 1.0 do not have published MD5 checksums, skipping check"; echo "" ;;
+        *) verify_sbt_jar "${sbt_jar}" ;;
+    esac
   }
 }
 
+verify_sbt_jar() {
+  local jar="${1}"
+  local md5="${jar}.md5"
+
+  if command -v md5sum > /dev/null 2>&1; then
+    md5_cmd="md5sum"
+  elif command -v md5 > /dev/null 2>&1; then
+    md5_cmd="md5"
+  else
+    echoerr "Could not find an MD5 command"
+    return 1
+  fi
+
+  download_url "$(make_url "${sbt_version}").md5" "${md5}" > /dev/null 2>&1
+  if echo "$(cat "${md5}")  ${jar}" | "${md5_cmd}" -c -; then
```

Comment:
This PR is looking really good now, thank you!

Two things:

First, how does a user without md5 _or_ md5sum progress?  With the current patch what they can do is download the jar and feed it to `-sbt-jar`.  Is that good enough?  Is it silly to have a `-sbt-jar-md5`?  🤔 let me know if you have any thoughts on this.

The second one is `md5sum -c -` works but `md5 -c -` isn't.  Here's what I've found works with `md5`:

```bash
09:22:42 $ if [ "$(md5 -q sbt-launch-1.2.8.jar)" == "$(cat sbt-launch-1.2.8.jar.md5)" ]; then echo rm; else echo fail; fi
rm
```